test/mpi/util/mtest_thread.c

```diff
@@ -9,6 +9,10 @@
    that disallow pthread_barrier_t and friends.
 */
```

Comment:
You might as well move these comments to where it applies. (you can do this before merging, no need interrupt tests).

Reply:
The second commit fixed it.